nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -221,15 +220,13 @@ def quantize_input(self, *inputs, wrapper, **kwargs):
             self.record(wrapper, 'input', inputs)
         return inputs
 
-    def quantize_weight(self, wrapper, **kwargs):
+    def quantize_weight(self, weight, wrapper, **kwargs):
```

Comment:
could you explain more about this change? `weight` can be obtained from `wrapper`, why pass it again?

Reply:
It's about the code readability. Since we already pass the weight to be quantized (`new_weight`) to quant_grad [here](https://github.com/microsoft/nni/blob/9cd42f61fbd4ea227b8935353a58b0472ac212ff/nni/compression/pytorch/compressor.py#L562), it is better to directly use it instead of obtaining it from wrapper. The developer can easily know that `quantize_weight` is a big op to simulate quantization and the op takes origin/bn-folded weight as input. I think using `wrapper.weight` will make it difficult to understand to structure of the training graph.